### DISCUSSION TITLE
Allow "admin" username to be defined for containers like local-ai

### DISCUSSION THREAD

**Discussion by ShapeShifter499** [nextcloud/all-in-one#3796] [Ideas] [answered]
I changed the "default admin" username in my Nextcloud AIO to something different before I saw this container. Can there be an added option to specify the admin user username?

Currently, the local-ai container doesn't work if the username is anything other than "admin" for the administrator account.  On my setup, a user named "admin" does not exist. 

Originally posted an issue here https://github.com/szaimen/aio-local-ai/issues/7 but opened up one here for more visibility.

**Comment by szaimen** [comment, marked as answer]
Hi, currently there are no plans to make this configurable.

However as a workaround you could try to check if recreating the default admin user works.

**Reply by ShapeShifter499** [reply]
So essentially, I'll have two admin users?  It defeats the purpose of changing the username in my case.

I migrated from a snap install where I had already named the administrator account with a unique username. So everything, including the folder, was renamed.

EDIT: I understand I could go back to just having one admin user,  but my concerns still stand

**Reply by ShapeShifter499** [reply]
I originally changed the username because I thought it would make it harder for remote attacks. This is an internet facing setup.

**Reply by ShapeShifter499** [reply]
I meant to get back to this earlier but the holidays got in the way. 


I went and added an admin user to my setup called "admin". I had to clear the volumes related to local-ai and reinstall the mastercontainer. But after doing so, it seems to work fine here. 


I guess as long as I have 2fa and a good password, it should be okay, I suppose.
